internal/node/validations.go

```diff
@@ -3,17 +3,24 @@ package node
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/pkg/errors"
 	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/aws/eks-hybrid/internal/kubelet"
 )
 
 const defaultStaticPodManifestPath = "/etc/kubernetes/manifest"
 
+const (
+	nodeValidationInterval = 30 * time.Second
+	nodeValidationTimeout  = 5 * time.Minute
```

Comment:
Are this random values or we have some reason behind them?

Reply:
they're random

Reply:
Those were random placeholders while I test. I'm going to change it to 5 second intervals and 1 minute timeout. Since it stops after 5 consecutive errors, 1 minute makes sense since it allows for up to 6 retries assuming there's no delay in getting nodes. 5 seconds should be enough time for transient issues to resolve without polling too frequently.

Reply:
If it turns out that's not enough, we can increase the duration in the future.